test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
+    border: 1px solid #888;
+    font-family: sans-serif;
     margin-bottom: 10px;
```

Comment:
I think that it is a "bad" idea in terms of backwards compatibility too.
For the record I didn't think of adding `margin-bottom` but `margin-top` starting from the second "row" a bit like [this](http://www.yelp.com/styleguide#section-arrange) (scroll to the bottom of the section/see arrange--wrap)